momento-sdk/src/main/java/momento/sdk/exceptions/WrappedSdkException.java

```diff
@@ -0,0 +1,8 @@
+package momento.sdk.exceptions;
+
+/** An SdkException that is a wrapper around another SdkException. */
+public class WrappedSdkException extends SdkException {
+  public WrappedSdkException(SdkException cause) {
```

Comment:
nit: couldn't you just add this as an alternate constructor for the main SdkException class?

Reply:
Well, that would be too easy. 🤦 I'll switch to that.